README.md

```diff
@@ -95,6 +95,7 @@ Documentation generated using [Doxygen](https://www.doxygen.nl/index.html). Run
 
 Pull requests are welcome. For major changes, please discuss in the Discord.
 
-Run `clang-format` on your code to ensure it meets the project's coding standards.
+Run `make format` to ensure it meets the project's coding standards.
+If some code don't need to be formatted add `// clang-format off` before and `// clang-format on` after the part of the code who need it
```

Comment:
If some code needs to avoid formatting place `// clang-format off` before the code and `// clang-format on` after.